docs/v3.x/getting-started/troubleshooting.md

```diff
@@ -45,20 +45,20 @@ With the release of the Strapi beta version a fundamental change occurred in tha
 
 Strapi has released the new Admin & Permissions (RBAC - Role based access control) that does allow for some degree of control over what users can access within the admin panel and includes some field level permissions. You can now also give roles specific permissions for things like content-types, single-types, plugins, and settings.
 
-When this new plugin release, there will be two versions:
+When this new plugin release, there is two versions:
 
 - Community Edition
 - Enterprise Edition
 
-By default, the Community Edition will include 3 pre-defined roles (Administrators, Editor, Author). Upgrading to the Enterprise Edition will unlock an unlimited number of roles. There will be certain other field level permission limitations based on the edition and we will be building a detailed guide as to what is included within the "Basic" vs "Advanced" RBAC features. To learn more about what is included as well as pricing please see our [pricing page](https://strapi.io/pricing).
+By default, the Community Edition includes 3 pre-defined roles (Administrators, Editor, Author). Upgrading to the Enterprise Edition will unlock an unlimited number of roles. There will be certain other field level permission limitations based on the edition and we will be building a detailed guide as to what is included within the "Basic" vs "Advanced" RBAC features. To learn more about what is included as well as pricing please see our [pricing page](https://strapi.io/pricing).
 
 ### Relations aren't maintaining their sort order
 
 With the components there is a hidden field called `order` that allows entries to maintain their sort, however with relations there is no such field. If you consider the typical count of of component entries vs relational based entries (in retrospect they function in the backend the same) there is generally a much higher number of relations. If relations were to have an `order` field applied to them as well it could cause significant performance degradation when trying to update the order, and likewise in the case where a relation could be attached to multiple entries it would be quite difficult to maintain the order.
 
 For the time being there is no recommended way to handle this automatically and instead it may be required for you to create custom controllers to handle this within your own project.
 
-### Why is my app's database and uploads resetting on Heroku
+### Why is my app's database and uploads resetting on PaaS (Heroku, DigitalOcean Apps, Google App Engine, ect)
```

Comment:
We should avoid having parenthesis in a title, even more writing "etc".
I think you can stop after "Paas", but mention Heroku, DigitalOcean Apps and Google App Engine in the paragraph below.

Reply:
Makes sense